test/e2e/run_seccomp_test.go

```diff
@@ -39,10 +39,7 @@ var _ = Describe("Podman run", func() {
 			//   127 + failed to connect to container's attach socket ... ENOENT
 			Expect(session.ExitCode()).To(BeNumerically(">=", 126), "Exit status using runc")
 		} else {
-			expect := "OCI runtime error: crun: read from the init process"
-			if IsRemote() {
-				expect = "for attach: crun: read from the init process: OCI runtime error"
-			}
+			expect := "read from the init process"
```

Comment:
I really dislike loose error-message checks, so I won't be approving this. I won't block, either.

However, please get rid of the temporary variable, it's no longer necessary.

Reply:
reverted to use the full message with `podmanTest.OCIRuntime`.

And added another fix on top of it when the OCI runtime is neither crun nor runc